primerpedia.js

```diff
@@ -62,4 +62,24 @@ function apiRequest(queryString){
 			}
 		}
 	});
-}
\ No newline at end of file
+}
+
+function getQueryVariable(variable) {
+	// http://stackoverflow.com/questions/2090551/parse-query-string-in-javascript
+    var query = window.location.search.substring(1);
```

Comment:
Mixing spaces and tabs here :) please follow the current code style and use tabs for indentation.